lib.es6.d.ts does not include typings for Headers.prototype.entries

<!-- BUGS: Please use this template. -->
<!-- QUESTIONS: This is not a general support forum! Ask Qs at http://stackoverflow.com/questions/tagged/typescript -->
<!-- SUGGESTIONS: See https://github.com/Microsoft/TypeScript-wiki/blob/master/Writing-Good-Design-Proposals.md -->

**TypeScript Version:**  2.2.1

**Code**

```ts
$ cat test.ts
const h = new Headers();

h.entries();
$ tsc test.ts
test.ts(3,3): error TS2339: Property 'entries' does not exist on type 'Headers'.
```

**Expected behavior:**

TypeScript does not complain about the `entries` property, since it is [part of the spec](https://developer.mozilla.org/en-US/docs/Web/API/Headers) and implemented by Firefox and Chrome.

**Actual behavior:**

TypeScript complains that the `entries` property does not exist.

--- a/src/lib/dom.iterable.d.ts
+++ b/src/lib/dom.iterable.d.ts
@@ -22,6 +22,7 @@ interface FormData {
 }
 
 interface Headers {
+    [Symbol.iterator](): IterableIterator<[String, String]>;
     /**
      * Returns an iterator allowing to go through all key/value pairs contained in this object.
      */

--- a/src/lib/dom.iterable.d.ts
+++ b/src/lib/dom.iterable.d.ts
@@ -21,6 +21,21 @@ interface FormData {
     [Symbol.iterator](): IterableIterator<string | File>;
 }
 
+interface Headers {
+    /**
+     * Returns an iterator allowing to go through all key/value pairs contained in this object.
+     */
+    entries(): IterableIterator<[string, string]>;
+    /**
+     * Returns an iterator allowing to go through all keys f the key/value pairs contained in this object.
+     */
+    keys(): IterableIterator<string>;
+    /**
+     * Returns an iterator allowing to go through all values of the key/value pairs contained in this object.
+     */
+    values(): IterableIterator<string>;
+}
+
 interface NodeList {
     /**
      * Returns an array of key, value pairs for every entry in the list

--- a/src/lib/dom.iterable.d.ts
+++ b/src/lib/dom.iterable.d.ts
@@ -22,7 +22,7 @@ interface FormData {
 }
 
 interface Headers {
-    [Symbol.iterator](): IterableIterator<[String, String]>;
+    [Symbol.iterator](): IterableIterator<[string, string]>;
     /**
      * Returns an iterator allowing to go through all key/value pairs contained in this object.
      */

